### ISSUE TITLE
reportUnknownArgumentType false positive assigning empty list to a subscript of a list

### ISSUE THREAD

**Issue by JonathanPlasse** [microsoft/pyright#7278]
**Describe the bug**
Assigning an empty list to a subscript of a list should not cause an error.

```console
error: Argument type is partially unknown
    Argument corresponds to parameter "__value" in function "__setitem__"
    Argument type is "list[Unknown]" (reportUnknownArgumentType)
```

**Code or Screenshots**

```python
a = [1, 2, 3]
a[1:] = []
```

**VS Code extension or command-line**
Pyright: v1.1.350

**Comment by erictraut** [comment]
Thanks for reporting the issue. It will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.351, which I just published.
